src/client/theme-default/components/VPNavBarTitle.vue

```diff
@@ -9,7 +10,7 @@ const { hasSidebar } = useSidebar()
 <template>
   <div class="VPNavBarTitle" :class="{ 'has-sidebar': hasSidebar }">
     <a class="title" :href="site.base">
-      <img v-if="theme.logo" class="logo" :src="theme.logo">
+      <VPImage class="logo" :image="theme.logo" />
```

Comment:
This class is also not present on final `img` tag.

Reply:
The issue in production also appears to happen because `class` (`dark`, `light`) isn't forwarded to `img`.

Reply:
Thanks for testing! It should be fixed now.